plugins/amdgpu/amdgpu_plugin.c

```diff
@@ -1795,7 +1796,12 @@ int amdgpu_plugin_restore_file(int id)
 		 * copy of the fd. CRIU core owns the duplicated returned fd, and amdgpu_plugin owns the fd stored in
 		 * tp_node.
 		 */
-		return dup(fd);
+		fd = dup(fd);
+		if (fd == -1) {
+			pr_perror("unable to duplicate the render fd");
+			return -1
```

Comment:
missing a semicolon here

Reply:
I've opened a pull request to build the amdgpu plugin in CI that should catch similar errors: https://github.com/checkpoint-restore/criu/pull/2276